jitouch/Jitouch/Gesture.m

```diff
@@ -1199,9 +1199,11 @@ static int gestureTrackpadMoveResize(const Finger *data, int nFingers, double ti
         if (step2 == 0) {
             if (firstTime) {
                 getMousePosition(&baseX, &baseY);
-                NSAutoreleasePool *pool = [[NSAutoreleasePool alloc] init];
-                [cursorWindow orderOut:nil];
-                [pool release];
+                dispatch_async(dispatch_get_main_queue(), ^{
+                    NSAutoreleasePool *pool = [[NSAutoreleasePool alloc] init];
+                    [cursorWindow orderOut:nil];
+                    [pool release];
```

Comment:
Can you use `@autoreleasepool` [blocks](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/MemoryMgmt/Articles/mmAutoreleasePools.html) instead of `NSAutoreleasePool`?

```suggestion
                    @autoreleasepool {
                        [cursorWindow orderOut:nil];
                    }
```